andela/eagle-bn-backend
# #168781804 email verification

### What does this PR do?
- user can verify their email before they merge
- documentation of the email verification
#### Description of Task to be completed?
usually, users need to verify their email. to make sure that those emails belong to them
#### How should this be manually tested?
create an account on sendgrid, and generate your api key
user can verify their email through post request by using:
`post:/api/v1/users/verify`
user can verify their email through email sent to them
`get:/api/v1/users/verify/:token`
#### Any background context you want to provide?
NAN
#### What are the relevant pivotal tracker stories?
[story](https://www.pivotaltracker.com/story/show/168781804)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/33803574/66580552-de548b00-eb7e-11e9-9956-350c2098aee2.png)

#### Questions:
NAN
